### PR TITLE
Add French translations, update README

### PR DESCRIPTION
**Pull request by florimondmanca**
Hi there,

I added French translations to the `django-simple-blog` app. Also updated the README to mention that the app is now available in English, Swedish and French.

The translations were tested on an example project with Django 1.11 and `LANGUAGE_CODE = 'fr-FR'`. See the project's `requirements.txt` below :

```
Django==1.11
django-el-pagination==3.0.1
django-markdown-deux==1.0.5
django-pagedown==0.1.3
-e git+https://github.com/florimondmanca/django-simple-blog.git@5675100aa073124e516b1a35e1ea7283de79c9a7#egg=django_simple_blog
django-simple-math-captcha==1.0.7
markdown2==2.3.5
pytz==2017.3
```

Screenshots from the French-translated example project below :

admin-home
admin-post

Let me know of any missing pieces!

P.S. : this will be my very first open source contribution! 👍

### CODE REVIEW

File: simpleblog/locale/fr_FR/LC_MESSAGES/django.po [hunk 1]
```diff
@@ -0,0 +1,98 @@
+# French translation of this blog.
+# Copyright (C) 2017
+# This file is distributed under the same license as the PACKAGE package.
```
Comment (drager) [inline simpleblog/locale/fr_FR/LC_MESSAGES/django.po]: `PACKAGE` should be `django-simple-blog`.

**Comment by drager** [comment]
Hey! Thank you so much for this pull request! It looks great, just one thing that I mentioned in the review. :)

Cool that this is your first pull request! Welcome to open source and thanks once again for the pull request!

**Comment by florimondmanca** [comment]
@drager Hey, thanks for the feedback! I fixed the PACKAGE keyword as requested (and refined the translations just a bit at the same time) and recompiled the messages.

**Comment by drager** [comment]
@florimondmanca: Great, thanks. I will merge it now :)

**Pull request merged**